### PR TITLE
Devcontainer

### LINKED ISSUE #242: Devcontainer
Can we have a `.devcontainer` like in Keras?

https://github.com/keras-team/keras/tree/master/.devcontainer

### PR DESCRIPTION
**Pull request by bhack**
resolve  #242

### CODE REVIEW

**Comment by LukeWood** [comment]
Thanks for the PR @bhack

File: .devcontainer/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ARG VARIANT
```
Comment (LukeWood) [inline .devcontainer/Dockerfile:1]: Any way I can test this?  I've never used .devcontainer - perhaps github codespaces is the easiest way?  Looks good from a birds-eye view.

Comment (bhack) [reply .devcontainer/Dockerfile:1]: You can test this "locally" with Vscode (CPU/GPU), locally with Codespace, or in the browser with Github with Codespace top right button in this page (CPU only instance)

Comment (LukeWood) [reply .devcontainer/Dockerfile:1]: I don't have VSCode - and I'm not in the codespaces beta.  For now this looks fine - if it works for you it looks good to me.

**Comment by LukeWood** [comment]
Thanks @bhack

File: .devcontainer/Dockerfile (hunk 1 above)
Comment (bhack) [reply .devcontainer/Dockerfile:1]: Ok but Codespace is not in beta.
Probably you have an Org account and you Org has not activated Codespaces.

With your personal Github account you will find the button (top right)

**Pull request merged**